openwisp_controller/geo/estimated_location/tests/tests.py

```
@@ -0,0 +1,998 @@
+                "Unable to create estimated location for device",
+                "Please assign/create a location manually.",
+            ]
+            _verify_notification(device3, messages, "error")
+
+    @mock.patch.object(config_app_settings, "WHOIS_CONFIGURED", True)
+    @mock.patch("openwisp_controller.config.whois.service.send_whois_task_notification")
+    @mock.patch(
+        "openwisp_controller.geo.estimated_location.tasks.send_whois_task_notification"
+    )
+    @mock.patch(
+        "openwisp_controller.config.whois.service.WHOISService.trigger_estimated_location_task"  # noqa E501
```

Ruff (0.15.0)

[warning] Unused blanket `noqa` directive

Remove unused `noqa` directive

(RUF100)